x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/inference/action/InferenceAction.java

```diff
@@ -47,7 +47,7 @@
 public class InferenceAction extends ActionType<InferenceAction.Response> {
 
     public static final InferenceAction INSTANCE = new InferenceAction();
-    public static final String NAME = "cluster:monitor/xpack/inference";
```

Comment:
@davidkyle just wanted to confirm that this is what we want to do here right? Changing it to internal?